### PR TITLE
Route all searches through Core

### PR DESCRIPTION
**Pull request by ecotopian**

### CODE REVIEW

File: Client/Frontend/Browser/OpenSearch.swift [hunk 1]
```diff
@@ -65,7 +66,7 @@ class OpenSearchEngine: NSObject, NSCoding {
      * Returns the search URL for the given query.
      */
     func searchURLForQuery(_ query: String) -> URL? {
-        return getURLFromTemplate(searchTemplate, query: query)
+        return URL.search(query: query)
```
Comment (vauxhall) [inline Client/Frontend/Browser/OpenSearch.swift:69]: Core strikes back 🎉

**Review by vauxhall — APPROVED** [review]

**Comment by vauxhall** [comment]
Why closing this?

**Comment by ecotopian** [comment]
> Why closing this?

oh that was by mistake.. I thought I merged it and my force push reopened it,, but that's apparently not the case 🙈 .. reopening

**Comment by ecotopian** [comment]
changes also included in https://github.com/ecosia/ios-browser/pull/41

**Pull request closed without merging**